src/main/topLevelPages/myDashboard/DashboardContext.tsx

```diff
@@ -0,0 +1,39 @@
+import React, { createContext, useState, useContext, ReactNode } from 'react';
+import { getCachedView, setViewToCache } from './dashboardUtil';
+
+const dashBoardViews = {
+  ACTIVITY: 'ACTIVITY',
+  SPACES: 'SPACES',
+} as const;
+
+interface DashboardContextProps {
```

Comment:
Let's move this and the `dashBoardViews` to a separate types file like we discussed before for new files, what do you say?

Reply:
I'd keep these as an exception, let's have this context encapsulated without having additional folder and a file. 
Otherwise, I'm following this types extraction convention in most places.